src/bomi/player/mainwindow_m.cpp

```diff
@@ -269,6 +269,19 @@ auto MainWindow::Data::connectMenus() -> void
     connect(play(u"chapter"_q)[u"next"_q], &QAction::triggered, p,
             [seekChapter] () { seekChapter(+1); });
 
+    connect(play[u"state"_q], &QAction::triggered, p, [=] () {
+        QString str;
+        str.append(_MSecToString(e.time()));
```

Comment:
Your original code didn't put miliseconds but this will. Is this intended? If not, you can give format argument "hh:mm:ss" to _MSecToString().